example/shchehelskyiho/second/task1.py

```diff
@@ -0,0 +1,32 @@
+'''
+Користувач вводить час у форматі `hh:mm(:ss(.ms))`, де дужки позначають необов'язковість.
+Напишіть функцію, що повертає кількість повних секунд, що пройшли з опівночі.
+Виведіть результат.
+У випадку помилкового вводу виведіть повідомлення про помилку.
+'''
+import re
+
+def validator(text):
```

Comment:
`validator` is better to be used as `get_date`, because all checks are common for all cases. And input text is the same always - `'Введіть час: '` . So I'd recommend to put it inside the function.